smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -132,16 +134,25 @@ public async Task Configure(string connectionString, string image, string hostna
                         cts.Token);
                 }
 
-                string args = $". {this.scriptDir}\\IotEdgeSecurityDaemon.ps1; Install-SecurityDaemon -Manual " +
-                              $"-ContainerOs Windows -DeviceConnectionString '{connectionString}' -AgentImage '{image}'";
-
-                this.proxy.ForEach(proxy => { args += $" -Proxy '{proxy}'"; });
+                string args;
+                if (this.isCleanInstall)
+                {
+                    args = $". {this.scriptDir}\\IotEdgeSecurityDaemon.ps1; Install-SecurityDaemon -Manual " +
```

Comment:
Add log using Console.WriteLine, e.g. Console.WriteLine("Clean install for iotedge .....").

Reply:
Done.